manifests/mpm/disable_mpm_event.pp

```diff
@@ -1,19 +1,28 @@
 # @summary disable Apache-Module event
 class apache::mpm::disable_mpm_event {
+  $event_command = ['/usr/sbin/a2dismod', 'event']
+  $event_onlyif = ['/usr/bin/test', '-e', shell_escape(join([$apache::mod_enable_dir, 'event.load'],'/'))]
```

Comment:
`onlyif` should be a nested array since it can take an array of commands.

Reply:
changed